src/components/Media/Video/Video.tsx

```diff
@@ -10,6 +10,9 @@ import './Video.scss';
 
 const b = block('media-component-video');
 
+export const qaIdOfSourceByDefault = 'qa-video-source';
+export const qaIdOfVideoByDefault = 'qa-video';
+
```

Comment:
Don't we need custom qa here too?

Reply:
I have changed them to roles

Reply:
UPDATE: Have talk to Andrey. Adding role is not good idea. Describe it test's readme